source/Calamari.Shared/Integration/Certificates/WindowsNative/WindowsX509Native.cs

```diff
@@ -69,14 +68,7 @@ internal static extern bool CryptAcquireCertificatePrivateKey(SafeCertContextHan
                                                                       [Out] out int dwKeySpec,
                                                                       [Out, MarshalAs(UnmanagedType.Bool)] out bool pfCallerFreeProvOrNCryptKey);
 
-        [DllImport("Crypt32.dll", SetLastError = true)]
```

Comment:
Shuffled down to put between the other unmanaged invocations that are currently only available in netfx for this PR